global.json

```diff
@@ -4,7 +4,7 @@
   },
   "sdk": {
     "allowPrerelease": false,
-    "version": "8.0.407",
+    "version": "8.0.408",
```

Comment:
The change is fine, but it's very odd that you were broken by a Windows update. Even patch versions of the .NET SDKs generally can be installed side-by-side - for example on my dev machine:
<img width="273" alt="image" src="https://github.com/user-attachments/assets/675693e9-02ce-4135-8660-0eec70d8b519" />

How are you installing the .NET SDK?

Reply:
I used the official installer, and once every few weeks, Windows Update uninstalls 8.0.x-1 and installs 8.0.x. (This is why I was a vocal proponent of keeping `"rollForward": "enable"` in the global.json file.)

.NET lets you install SDKs side by side, but you have to do it manually.

Reply:
The problem is - you're going to get broken either way by the package lock files (see https://github.com/dotnet/sdk/issues/38200).

I would look into why you're getting auto-upgraded - it feels super strange to me that WindowsUpdate is removing older versions of the SDK when they are designed to be run side-by-side; I've never observed this behavior on Windows. Are you sure you're installing from MSI packages under https://dotnet.microsoft.com/en-us/download/dotnet/8.0 and not e.g. via winget?

Reply:
For me, the latest installed SDK version seems to be auto upgraded by the VS installer. It leaves the older ones untouched, though.

Reply:
I believe the replacements are the intended behavior. From [the .NET blog entry announcing this feature](https://devblogs.microsoft.com/dotnet/net-core-updates-coming-to-microsoft-update/):

> In the .NET Core case, major/minor (feature) updates always install side-by-side. Only monthly servicing updates install in-place and replace previous servicing updates. We are delivering .NET Core servicing updates monthly without issues. .NET Core servicing updates have evolved from fully side-by-side installs to patches that replace earlier versions with “runtime roll-forward”, so once an update is deployed apps can use the latest (serviced) runtime version automatically.